docs/Docs_Manual_QA.md

```diff
@@ -13,3 +13,5 @@ XXX Test Passed: <link to the test results in your GitHub repository>
 </br>
 XXX Test Failed: <link to the test results in your GitHub repository>, <link to the issue>
 </br>
+
+AI_WORKFLOW_SECTION (DEV_ENV) Test Passed: https://github.com/truesoni/hacktoberfest2023/blob/main/mindsdb/issues_7516/Testing.md
```

Comment:
replace AI_WORKFLOW_SECTION (DEV_ENV) with Create Project using Python
The images do not display properly
![image](https://github.com/mindsdb/mindsdb/assets/32901682/4ebc3f34-3a98-4763-b8a7-9cb4d124bac4)